rosetta-cli-conf/testnet/config_erc20.json

```diff
@@ -1,6 +1,6 @@
 {
   "network": {
-    "blockchain": "Avalanche",
+    "blockchain": "Camino",
     "network": "Fuji"
```

Comment:
and also the `network` here?

Reply:
fixed

Reply:
tbh I don't know if it should be columbus or Columbus or it doesn't matter